oat-sa/extension-tao-itemqti
# Fix/CON-238/reset settings in media after changing source

Related to: https://oat-sa.atlassian.net/browse/CON-238

Fixed reset of width in Media Interaction and object after changing source.

How to test:

- go to Item Authoring
- create Media Interaction and A-block with object contained video with width 50% for example
- save go to Items page
- return to item Authoring
- change source in Media Interaction(to audio) and object(to image)
- width should be reseted
